website/docs/rbs-support.md

```diff
@@ -198,6 +198,9 @@ of being able to advocate for using this syntax more widely.
 
 ### Runtime checking is a feature
 
+> Note that runtime checking of RBS signatures is not yet implemented, so type
+> safety is reduced.
```

Comment:
Could you rephrase this to avoid implying that we're planning to implement runtime support for RBS? I'm not sure we've discussed any paths forward here, and  I don't want to give the false impression that this is on our roadmap. I think removing `yet` would be sufficient.
```suggestion
> Note that runtime-checking of RBS signatures is not implemented, so type
> safety checking is reduced.
```

Reply:
Updated, but now it's a little odd that the next paragraphs discuss runtime checking.

Reply:
You could move it underneath that first paragraph to improve the flow a bit?